.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+Operating System Version:
```

Comment:
You need to isolate the actual code, please. You can't push the entire `popcorn-official:development` commit list here